aws/data_source_aws_organizations_organization.go

```diff
@@ -156,75 +156,73 @@ func dataSourceAwsOrganizationsOrganizationRead(d *schema.ResourceData, meta int
 	d.Set("master_account_email", org.Organization.MasterAccountEmail)
 	d.Set("master_account_id", org.Organization.MasterAccountId)
 
-	if aws.StringValue(org.Organization.MasterAccountId) == meta.(*AWSClient).accountid {
```

Comment:
Basically this line is all that has been removed, i guess. The diff is a bit weird.